.github/workflows/Manager-CI.yaml

```diff
@@ -35,6 +35,10 @@ jobs:
         uses: ./.github/actions/java-maven-setup
         with:
           cache-key-prefix: ${{ runner.os }}
+      - name: Re-Checkout  # since Java And Maven Setup step is checking out the main branch, we have to checkout the pull request branch again
```

Comment:
If this is the case, is it possible to just move the checkout _after_ the configuration of java/maven?

Reply:
No, because the `Java and Maven Setup` is using local action so it requires a code to be already checked out

Reply:
Ok I don't understand the reason but fine fine for me :)